packages/workspace/src/browser/workspace-service.ts

```diff
@@ -48,7 +48,10 @@ export class WorkspaceService implements FrontendApplicationContribution {
 
     @postConstruct()
     protected async init(): Promise<void> {
-        const rootUri = await this.server.getRoot();
+        let rootUri = await this.server.getRoot();
+        if (rootUri && rootUri.endsWith("/")) {
```

Comment:
How about to move it into `toValidRoot` that it works as well for other cases?

Reply:
Done